### ISSUE TITLE
Updating line chart without animation

### ISSUE THREAD

**Issue by fabulousduck** [stsrki/Blazorise#148]
I currently have a dataset that updates every 5 seconds. Is it possible to disable the animation of the chart coming up so i can keep live updating it ?

**Comment by stsrki** [comment]
I think it can be done by setting the animation `duration` to zero. You can find the options on the **charts.js** [documentation](https://www.chartjs.org/docs/latest/configuration/animations.html)

Also: https://stackoverflow.com/questions/21389341/disable-animation-with-charts-js

**Comment by fabulousduck** [comment]
I have tried setting this in the `LineChartDataSet` from the example, but it tells me that there is no definition for it. Or do i need to set this value somewhere else ?

**Comment by stsrki** [comment]
There are two ways to set the options for charts.

1. By setting the `Options` attribute. Unfortunately this does not yet have an option to set the animation.
2. By setting the `OptionsJsonString` attribute. This is introduced as a workaround for faulty json serializer in older versions of Blazor.

You can go with the second option and send the full json string to the chart. Just keep in mind that OptionsJsonString will be removed eventually once the json serializer is fixed. 

I will add Animation property to Options. It will be done in v0.8.1

**Comment by fabulousduck** [comment]
It seems that `OptionsJsonString` is private as i am getting the following error

```
BaseChart<LineChartDataset<double>, double, LineChartOptions>.OptionsJsonString' is inaccessible due to its protection level
```

**Comment by stsrki** [comment]
Just set it like this for now `<LineChart @ref="lineChart" OptionsJsonString="@yourOptionsJson" />`

It's private because of one more Blazor constraint("feature") from the past. It will be made public in future versions. Welcome to the fast moving Blazor development :)

**Comment by fabulousduck** [comment]
that did the trick! thanks for helping out so fast :)

**Comment by stsrki** [comment]
Will be released with 0.8.1
